LeaderboardBackend/Models/Entities/ApplicationContext.cs

```diff
@@ -41,7 +41,7 @@ public ApplicationContext(DbContextOptions<ApplicationContext> options, IClock c
     {
         _clock = clock;
         ChangeTracker.Tracked += AddCreationTimestamp;
-        ChangeTracker.Tracked += SetUpdateTimestamp;
+        ChangeTracker.StateChanged += SetUpdateTimestamp;
```

Comment:
This change should already be on main. Do you need to do another rebase?

Reply:
Guess so? I already did a rebase; guess it didn't go through or something.

Reply:
Rebased and it's gone.